src/components/image/index.tsx

```diff
@@ -233,6 +236,7 @@ class Image extends PureComponent<Props, State> {
           aspectRatio && {aspectRatio},
           !useImageInsideContainer && margins,
           useImageInsideContainer && styles.containImage,
+          !cover && sizeProps && {width, height},
```

Comment:
I don't think the `cover` is needed here, if the user makes a mistake they can fix it (this will be bugged without an overlay as well)

Reply:
Also, personally I prefer to remove the extra `const sizeProps`.
Maybe just `{width, height}` or `(width || height) && {width, height}`?

Reply:
1) If we know passing width and height not working on 'cover' prop, why not condition it? Why make the user figure it out (or not) by themseves? 
2) What do you mean by 'this will be bugged without an overlay as well'? This solution works whether you pass overlay or not...
3) Done

Reply:
I think I meant that if the user adds width and forgets height (or vice versa, or adds a width with cover etc) they will have bugs that they will need to fix.

Reply:
Yes, that's for sure